person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
```

Comment:
- [ ] OPTIONAL
As a best ruby practice, kindly prefer _require_relative_ over _require_ when dealing with class files from the same source directory